Bot occasionally throws a unexpected mimetype error

User emailed me with the following error to investigate:

`ERROR [2022-03-20 16:09:51] [ERROR] [ENDPOINT] [OpenSea] API response was invalid 0, message='Attempt to decode JSON with satellites.py:41 unexpected mimetype: text/html; charset=utf-8', url=URL('https://api.opensea.io/api/v1/collection/skulltoonsbytheodoru').`

--- a/cogs/satellites.py
+++ b/cogs/satellites.py
@@ -33,7 +33,7 @@ async def get_opensea_floor_price(url: str, alias: str) -> Union[FloorPrice, boo
     try:
         async with aiohttp.ClientSession() as session:
             async with session.get(url) as response:
-                content = await response.json()
+                content = await response.json(content_type=None)
                 if content:
                     price = content['collection']['stats']['floor_price']
                     return FloorPrice(source=source, price=f"{price} ETH", project=alias)

--- a/cogs/satellites.py
+++ b/cogs/satellites.py
@@ -39,6 +39,7 @@ async def get_opensea_floor_price(url: str, alias: str) -> Union[FloorPrice, boo
                     return FloorPrice(source=source, price=f"{price} ETH", project=alias)
     except Exception as error:
         log.error(f"[ENDPOINT] [{source}] API response was invalid {error}.")
+        return False
     log.error(f"[ENDPOINT] [{source}] API request did not return the expected response: {content}.")
     return False
 

